Sources/SwiftOperators/OperatorTable+Folding.swift

```diff
@@ -475,6 +475,7 @@ extension OperatorTable {
     ) {
       self.opPrecedence = opPrecedence
       self.errorHandler = errorHandler
+      super.init(viewMode: .sourceAccurate)
```

Comment:
Do we really only want to fold source accurate nodes here? I would think of sequence folding as a semantic operation and thus think that it might be better off folding the fixed up tree.

Reply:
Ideally we wouldn't be folding cat all if there's any errors 😅. Let's say there's a bunch of unexpected nodes, do we expect folding to remove them all?

Reply:
Ah, except we won't remove them (thanks @rintaro). So I suppose I don't really have a preference here. I can do fixed up 🤷